docs/compression_algorithms/CompressWeights.md

```diff
@@ -60,6 +60,17 @@ nncf_dataset = nncf.Dataset(data_source, transform_fn)
 compressed_model = compress_weights(model, mode=CompressWeightsMode.INT4_SYM, ratio=0.8, dataset=nncf_dataset)
 ```
 
+- Accuracy of the 4-bit compressed models also can be improved by using AWQ algorithm over data-based mixed-precision algorithm. It is capable to equalize some subset of weights to minimize difference between
+original precision and 4-bit.
+Below is the example how to compress 80% of layers to 4-bit integer with a default data-based mixed precision algorithm and AWQ.
+It requires to set `awq` to `True` additionally to data-based mixed-precision algorithm.
+
+```python
+from nncf import compress_weights, CompressWeightsMode, Dataset
+nncf_dataset = nncf.Dataset(data_source, transform_fn)
```

Comment:
As in the case with data-driven compression (HAWQ, etc.), such an example of `dataset` does not make any sense because this is not how it looks in practice.  My suggestion is to remove the two first lines or make it more relevant (so that the code can be copy-pasted).

Reply:
@AlexKoff88 please look at new example.